deployments/charts/kuma/crds/kuma.io_meshgatewayconfigs.yaml

```diff
@@ -51,6 +51,26 @@ spec:
                 description: Resources specifies the compute resources for the proxy
                   container. The default can be set in the control plane config.
                 properties:
+                  claims:
```

Comment:
what is the reason for including it in this PR?

Reply:
I is generated automatically  during `make check` after upgrading: `sigs.k8s.io/controller-runtime` and without upgrading controller-runtime project does not compile